### PR TITLE
add search with algolia

### PR DESCRIPTION
**Pull request by mykiwi**

### CODE REVIEW

**Comment by tgalopin** [comment]
This should not be merged until we have a proper idea on how to implement the front part.

File: app/Resources/views/base.html.twig [hunk 1]
```diff
@@ -379,9 +377,52 @@
         <script type="text/javascript" src="{{ webpack_asset('kernel.js') }}"></script>
         <script type="text/javascript" src="{{ asset('js/cookieconsent.min.js') }}"></script>
         {% block javascripts '' %}
+        <script src="https://cdn.jsdelivr.net/autocomplete.js/0/autocomplete.min.js"></script>
```
Comment (mykiwi) [inline app/Resources/views/base.html.twig]: All this part is temporary. Like @tgalopin said, we need the front part.

File: app/Resources/views/components/search.html.twig [hunk 2]
```diff
@@ -3,3 +3,52 @@
     <path d="M9.9263,0 C4.4443,0 0.0003,4.444 0.0003,9.926 C0.0003,15.408 4.4443,19.853 9.9263,19.853 C15.4083,19.853 19.8523,15.408 19.8523,9.926 C19.8523,4.444 15.4083,0 9.9263,0 M9.9263,1 C14.8483,1 18.8523,5.004 18.8523,9.926 C18.8523,14.848 14.8483,18.853 9.9263,18.853 C5.0043,18.853 1.0003,14.848 1.0003,9.926 C1.0003,5.004 5.0043,1 9.9263,1" fill="#333333"></path>
     <path class="stroke" d="M16.6294,16.7087 L22.6294,22.6357" stroke="#333333" stroke-linecap="round"></path>
 </svg>
+
```
Comment (mykiwi) [inline app/Resources/views/components/search.html.twig]: Here too.

File: app/Resources/views/base.html.twig [hunk 3]
```diff
@@ -379,9 +377,52 @@
...
         <script type="text/javascript">
             on(window, 'load', function() {
-                Kernel.boot('{{ app.debug ? 'debug' : assets_hash }}', {{ sentry_dsn ? ('\''~sentry_dsn~'\'')|raw : 'null' }});
+                Kernel.onLoad(function () {
+                    var client = algoliasearch("X2JCF5YMLQ", '8231b5a0919abf1f3019016b9606f5b4');
```
Comment (hhamon) [inline app/Resources/views/base.html.twig]: Hardcoded credentials???

**Comment by mykiwi** [comment]
It's not prod credentials, but yes I removed them. They will be disabled after merging this PR.

**Comment by tgalopin** [comment]
Included in https://github.com/EnMarche/en-marche.fr/pull/864

**Pull request closed without merging**